src/mqueryfront/src/Navigation.js

```diff
@@ -33,59 +34,68 @@ function Navigation(props) {
 
     return (
         <nav className="navbar navbar-expand-lg navbar-dark bg-dark mb-3">
-            <Link className="navbar-brand" to={"/"}>
-                <Logo width="150" height="100%" />
-            </Link>
-            <button
-                className="navbar-toggler"
-                type="button"
-                data-toggle="collapse"
-                data-target="#navbarSupportedContent"
-                aria-controls="navbarSupportedContent"
-                aria-expanded="false"
-                aria-label="Toggle navigation"
-            >
-                <span className="navbar-toggler-icon" />
-            </button>
+            <div className="container-fluid">
```

Comment:
This is a bit unrelated to the current PR - new bootstrap requires navbar contents to be contained within a conteiner-fluid div